### ISSUE TITLE
[Research] Familiarize team with OOOOOOOO X 

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#68443]
I have written OOOOOOOO X so many times I've forgotten what the actual Acronym is. Is it OOUX?

**Comment by katelyncai** [comment]
> I have written OOOOOOOO X so many times I've forgotten what the actual Acronym is. Is it OOUX?

Yes, it is actually OOUX which stands for Object-Oriented UX :)

**Comment by katelyncai** [comment]
Today we finished our [OOUX workshop series](https://docs.google.com/presentation/d/1ZIoOl_Hsqzp-9cEZn_JAOh0-ZBqmh01PyUOvQZbc2vI/edit?usp=sharing) with session 5, focused on attributes. We drafted [an object map](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1696288098267/8a2b80c2bd5c38b8fc7041a384edf5c7237b5f7b?sender=ube02cc3d1108083a94f75471) (unfinished/WIP) for the EZR form. @IMHieuVo had the great suggestion of possibly applying OOUX and the ORCA process to our work on Registration in the new year. Looking forward to revisiting our team's use of OOUX in 2024!
